scripts/build/collect_symbols.sh

```diff
@@ -57,7 +57,11 @@ if [[ ! -d "${AWS_LC_DIR}" ]]; then
 fi
 
 function filter_symbols() {
+if [[ "${CRATE_NAME}" =~ fips ]]; then
+  grep -E '^\w*$' | grep -v -E "^bignum_" | grep -v "curve25519_x25519" | grep -v "edwards25519_" | grep -v "p256_montj" | grep -v "p384_montj" | grep -v "p521_montj" | grep -v "p521_j" | grep -v "jent_" | grep -v "fips_" | grep -v "fips_check"
```

Comment:
style nit: you might consider making this pipe-per-line to ease readability and adding/removing clauses

```
grep -E '$pattern' \
    | grep -v -E foo \
    | grep -v bar \
...
```